docs/api/unified_api/custom_paths.rst

```diff
@@ -0,0 +1,27 @@
+Custom Paths in new Api
+=======================
+
+All path in new API have the same prefix structure: ``/api/v2/admin/`` or ``/api/v2/shop/``
+
+When you adding a new path to api resource configuration, you should remember to add also proper prefix.
+
+You can declare entire path for each operation (without ``/api/v2/`` this part is configured globally):
+
+.. code-block:: xml
+
+    <collectionOperation name="admin_get">
+        <attribute name="method">GET</attribute>
+        <attribute name="path">admin/orders</attribute>
+    </collectionOperation>
+
+or you can add proper prefix for all path in chosen resource:
+
+.. code-block:: xml
+
+    <attribute name="route_prefix">shop</attribute>
+
+.. note::
+
+    In some situations you may need add path with custom structure, in this cases you should configure it in security file (``ApiBundle/test/config/security.yaml``)
```

Comment:
```suggestion
    In some situations you may need to add a path with custom structure, in this case you should configure it in security file (``ApiBundle/test/config/security.yaml``)
```
or 'those cases' in plural form. But looking at context I would prefer this case ( just like in suggestion)